src/redux/api/backendApi.js

```diff
@@ -0,0 +1,13 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { API_BACKEND } from '../../helpers/Api';
+
+export const backendApi = createApi({
+  reducerPath      : 'backendApi',
+  keepUnusedDataFor: 10,
```

Comment:
10 seconds?

Reply:
If user will click the "Back" button in 10 sec (or will return to prev parameters in any other way) - the result will still be in cache

Reply:
Just comment the number with // seconds